composer.json

```diff
@@ -6,6 +6,7 @@
     "require": {
         "symfony/browser-kit": "*",
         "symfony/css-selector": "*",
-        "symfony/phpunit-bridge": "*"
+        "symfony/phpunit-bridge": "*",
+        "phpunit/phpunit": "*"
```

Comment:
alpha ordering rulez

Reply:
or not, because the order drives the order of recipes, right?

Reply:
I updated it. I think it won't matter, but after we merge, I'll double-check that. `phpunit/phpunit` and `phpunit-bridge` both share a few files (like `tests/bootstrap.php`) but ordering doesn't cause a problem there. And, thanks to the now-merged recipe PR, `phpunit/phpunit` no longer has a post-install text, so no problem there either.